### PR TITLE
Add Trillium menu from DiningStack

### PR DESCRIPTION
**Pull request by ilikesymmetry**
Access hardcodedMenus.json from DiningStack.  Sets menu for trillium in graphql while looping through cornell dining's json.

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 CORNELL_INSTITUTION_ID = '73116ae4-22ad-4c71-8ffd-11ba015407b1'
 GET_URL = 'https://services.get.cbord.com/GETServices/services/json'
 GIT_CONTENT_URL = 'https://raw.githubusercontent.com/cuappdev'
+STATIC_MENUS_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/hardcodedMenus.json'
```
Comment (young-k) [inline src/constants.py]: u already know, alphabetize

**Review by young-k — APPROVED** [review]
nifty nifty

File: src/data.py [hunk 2]
```diff
@@ -253,3 +258,7 @@ def format_time(start_time, end_time, start_date):
   new_end = "{}:{}:{}".format(end_date, end_hour, end_minute)
 
   return [new_start, new_end]
+
+def trillium_menu():
```
Comment (young-k) [inline src/data.py]: can we call this `get_trillium_menu` instead?

File: src/constants.py (hunk 1 above)
Comment (ilikesymmetry) [reply src/constants.py]: AGHHHH

File: src/data.py (hunk 2 above)
Comment (young-k) [reply src/data.py]: @cswenberg this 1 too

**Comment by ilikesymmetry** [comment]
@young-k

**Review by young-k — APPROVED** [review]
beautiful

**Pull request merged**